projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-details.service.ts

```diff
@@ -33,4 +40,14 @@ export class OrderDetailsService {
       switchMap(() => this.userOrderService.getOrderDetails())
     );
   }
+
+  get CancellationReturnRequestInputs(): CancellationReturnRequestEntryInput[] {
```

Comment:
I don't think this should be capitalized

Reply:
Yes!